src/test/resources/testscripts/DeleteWorkspace.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+## This script deletes the current workspace after running an integration test.
+
+terra status
+
+# delete any resources
+terra resources list --format=json > resources.json
+jq -c '.[]' resources.json | while read i; do
+  name=$(echo $i | jq -r '.metadata.name');
+  echo "deleting workspace resource $name"
+  terra resources delete --name=$name;
+done
+
+# delete the workspace
```

Comment:
is there only ever one workspace active now?

Reply:
Yes, there's only ever one active workspace. This hasn't changed recently. You can have access to multiple workspaces, but when you type e.g. `terra resources list` the CLI has to know which workspace you mean to execute that in.